conda_mirror/conda_mirror.py

```diff
@@ -682,8 +682,12 @@ def main(upstream_channel, target_directory, temp_directory, platform,
                 channel=channel,
                 platform=platform,
                 file_name=package_name)
-            _download(url, download_dir)
-            summary['downloaded'].add((url, download_dir))
+            try:
+              _download(url, download_dir)
+              summary['downloaded'].add((url, download_dir))
+            except:
+              logger.error('Unexpected error: %s. Aborting download.', sys.exc_info()[0])
```

Comment:
Maybe rendering the full stack trace with `logger.exception('Unexpected error: Aborting download')` would be better here?